examples/manipulate_sys_nice.rs

```diff
@@ -50,11 +50,17 @@ fn main() {
 
 fn renice(prio: libc::c_int) -> libc::c_int {
     // This is not proper logic, as it does not drain errno.
+    #[cfg(target_arch = "mips")]
+    return unsafe { libc::setpriority(libc::PRIO_PROCESS as i32, 0, prio) };
```

Comment:
And as we are repeating this in different place, I'd just define a conditional `PROC_ID` constant and just use that.